tsk/auto/auto_db.cpp

```diff
@@ -107,10 +108,16 @@ void TskAutoDb::setAddUnallocSpace(bool addUnallocSpace)
     setAddUnallocSpace(addUnallocSpace, -1);
 }
 
-void TskAutoDb::setAddUnallocSpace(bool addUnallocSpace, int64_t chunkSize)
+void TskAutoDb::setAddUnallocSpace(bool addUnallocSpace, int64_t minChunkSize)
+{
+    setAddUnallocSpace(addUnallocSpace, minChunkSize, -1);
+}
+
+void TskAutoDb::setAddUnallocSpace(bool addUnallocSpace, int64_t minChunkSize, int64_t maxChunkSize)
```

Comment:
Just realized we can remove the bool here.  If they want to disable it, they should call the method that takes no size arguments.  It's too late to change the existing one (that just takes one size).